http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -118,11 +119,18 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
                                                                                        kubernetesNamespace
       )
 
+      maybeCromwellDatabaseNames <- maybeCreateCromwellDatabases(app,
```

Comment:
Still reading/understanding code but am a bit confused about these functions. 

- `maybeCreateWsmIdentityAndDatabase` only acts on WDS app type and returns a KSA + DB name.
- `maybeCreateCromwellDatabases` only acts on Cromwell app type and creates several databases, but no KSA.

Questions:
1. Do the Cromwell databases require KSA(s) with which to install Workload Identity, so the containers can access the DBs?
2. Is there any way to have a more general purpose method that handles these various permutations (app type, database enabled or not, KSA needed or not)? Instead of adding more `maybeCreateXyz` methods..

Reply:
`maybeCreateWsmIdentityAndDatabase` creates a _WSM_ identity and WSM creates a corresponding KSA with the convention that the name of the identity is the same as the KSA.

I feel like there should be an effort to standardize on a flexible config model. Then we can talk about general purpose. I don't really see how to be more generic in the current structure